### PR TITLE
[Console] fix typo in phpdoc

### PR DESCRIPTION
**Pull request by Tobion**
| Q             | A
| ------------- | ---
| Branch?       | 4.1 (#26847)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Console/Output/OutputInterface.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function write($messages, $newline = false, $options = 0);
     /**
      * Writes a message to the output and adds a newline at the end.
      *
-     * @param string|iterable $messages The message as an iterable of lines of a single string
+     * @param string|iterable $messages The message as an iterable of lines or a single string
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Output/OutputInterface.php]: s/lines/string/ also?

Comment (Tobion) [reply src/Symfony/Component/Console/Output/OutputInterface.php]: In this case the strings are output as lines (printLn). But I don't mind to change it as well if you want.

Comment (chalasr) [reply src/Symfony/Component/Console/Output/OutputInterface.php]: that'd be consistent IMHO

Comment (ogizanagi) [reply src/Symfony/Component/Console/Output/OutputInterface.php]: I agree with this suggestion too.

Comment (nicolas-grekas) [reply src/Symfony/Component/Console/Output/OutputInterface.php]: Re-reading, I'd prefer to keep "lines" above, don't you think?

Comment (chalasr) [reply src/Symfony/Component/Console/Output/OutputInterface.php]: I think I prefer talking about php types only, but no strong opinion really.

**Comment by nicolas-grekas** [comment]
Thank you @Tobion.

**Pull request merged**